com.unity.cinemachine/Tests/Runtime/CinemachineBrainSetupTest.cs

```diff
@@ -0,0 +1,48 @@
+#if UNITY_EDITOR //Selection.activeGameObject
+using System.Collections;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Cinemachine.Tests.Editor
```

Comment:
You're using the Editor namespace in the runtime assembly.  Is this an editor test or a runtime test?

![image](https://user-images.githubusercontent.com/6424658/225612009-113c82e6-f407-46c7-bb82-bea674a8c7f7.png)

Reply:
If you look at StateDrivenCameraTests you'll see that it's in the proper runtime namespace, and that it has #if UNITY_EDITOR to ensure that it only runs in the context of editor.  You should do it that way.

Reply:
Got it I missed it thanks for catching it.